src/test/java/uk/gov/hmcts/reform/cwrdapi/service/impl/StaffRefDataServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.cwrdapi.service.impl;
+
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.cwrdapi.domain.UserType;
+import uk.gov.hmcts.reform.cwrdapi.repository.UserTypeRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class StaffRefDataServiceTest {
+
+    @Mock
+    private UserTypeRepository userTypeRepository;
+
+    @InjectMocks
+    private StaffRefDataServiceImpl staffRefDataService;
+
+    @Test
+    @SuppressWarnings("unchecked")
+    void testFetchUserType_All() {
+        when(userTypeRepository.findAll())
+                .thenReturn(prepareUserTypeResponse());
+        var userTypes = staffRefDataService
+                .fetchUserTypes();
+        verifyAllUserTypes(userTypes);
```

Comment:
assert the expected count and their all content

Reply:
@rajkatla-hmcts - Added